### PR TITLE
Fix HandlersTimeProfiler example rendering

### PR DESCRIPTION
**Pull request by trsvchn**
Fixes https://github.com/pytorch/ignite/pull/1398#issuecomment-719936591

Description:

Some css  tweaks related to docs example rendering

Check list:
* [ ] New tests are added (if a new feature is added)
* [ ] New doc strings: description and/or example code are in RST format
* [ ] Documentation is updated (if required)

### CODE REVIEW

**Comment by trsvchn** [comment]
I've tried to fit 100% zoom

File: docs/source/_templates/_static/css/ignite_theme.css [hunk 1]
```diff
@@ -143,3 +143,7 @@ article.pytorch-article table.longtable.docutils.align-default tbody td p {
   margin-block-start: 0.1em;
   margin-block-end: 0.1em;
 }
+
+article.pytorch-article div#ignite-contrib-handlers.section div#time-profilers.section dl.py.class dd dl.py.method dd div.highlight-text.notranslate div.highlight pre {
+  font-size: 62%;
```
Comment (vfdev-5) [inline docs/source/_templates/_static/css/ignite_theme.css:148]: Maybe, 75% will be enough and not that small :)

**Review by vfdev-5 — COMMENTED** [review]
Thanks for the PR, @trsvchn

File: docs/source/_templates/_static/css/ignite_theme.css (hunk 1 above)
Comment (trsvchn) [reply docs/source/_templates/_static/css/ignite_theme.css:148]: the problem is 62% is the highest acceptable value to fit line, 75 will break the lines

Comment (trsvchn) [reply docs/source/_templates/_static/css/ignite_theme.css:148]: 75%
Screenshot from 2020-10-31 19-37-12

Comment (trsvchn) [reply docs/source/_templates/_static/css/ignite_theme.css:148]: There is another idea, to trim that example a little, but keep all important information:

```diff

            -------------------------------------  -------------------  -------------- ...
            Handler                                Event Name                 Total(s)
            -------------------------------------  -------------------  --------------
            run.<locals>.log_training_results      EPOCH_COMPLETED            19.43245
            run.<locals>.log_validation_results    EPOCH_COMPLETED             2.55271
            run.<locals>.log_time                  EPOCH_COMPLETED             0.00049
            run.<locals>.log_intermediate_results  EPOCH_COMPLETED             0.00106
            run.<locals>.log_training_loss         ITERATION_COMPLETED           0.059
            run.<locals>.log_time                  COMPLETED             not triggered
            -------------------------------------  -------------------  --------------
            Total                                                             22.04571
            -------------------------------------  -------------------  --------------
            Processing took total 11.29543s [min/index: 0.00393s/1875, max/index: 0.00784s/0,
             mean: 0.00602s, std: 0.00034s]
            Dataflow took total 16.24365s [min/index: 0.00533s/1874, max/index: 0.01129s/937,
             mean: 0.00866s, std: 0.00113s]

``` 

with such modification we can increase font-size from 62 to 72

Comment (trsvchn) [reply docs/source/_templates/_static/css/ignite_theme.css:148]: or 69% to fit that `...`

**Comment by trsvchn** [comment]
@vfdev-5 Wait! Look what I've found!!
Screenshot from 2020-10-31 20-11-06

A scrollbar!!

**Comment by vfdev-5** [comment]
Awesome! Let's add the scrollbar and wont bother with the size 👍

**Comment by vfdev-5** [comment]
What do you think if we enable the scrollbar for all sections ?

**Comment by trsvchn** [comment]
Are you thinking about enabling it globally for a such code example or just for the `contrib.handlers`?

**Comment by vfdev-5** [comment]
Yes, I was thinking about enabling it globally for all code examples.

**Comment by trsvchn** [comment]
Yes, I think it would be great to have a such feature! I am going to try it!

File: docs/source/_templates/_static/css/ignite_theme.css [hunk 2]
```diff
@@ -144,6 +144,6 @@ article.pytorch-article table.longtable.docutils.align-default tbody td p {
   margin-block-end: 0.1em;
 }
 
-article.pytorch-article div#ignite-contrib-handlers.section div#time-profilers.section dl.py.class dd dl.py.method dd div.highlight-text.notranslate div.highlight pre {
```
Comment (trsvchn) [inline docs/source/_templates/_static/css/ignite_theme.css]: I think this should help

**Review by vfdev-5 — APPROVED** [review]
Thanks a lot @trsvchn !

**Pull request merged**